run-postgres.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+echo 'Starting database...'
+
+docker run \
+  --name ucp-postgres \
+  --env POSTGRES_PASSWORD=ucp-rulez \
+  --publish 5432:5432 \
+  --init \
+  --tty \
+  postgres
```

Comment:
should this be: "ghcr.io/radius-project/mirror/postgres:latest"

Reply:
Added this to the makefile since this script is being deleted.